src/main/java/io/cryostat/graphql/ActiveRecordings.java

```diff
@@ -322,6 +323,41 @@ public RecordingOptions asOptions() {
         }
     }
 
+    @Blocking
+    @Transactional
+    @Description(
+            "Updates the metadata labels for an existing Flight Recorder recording specified by the"
+                    + " recording ID.")
```

Comment:
I think just `Updates the metadata labels for an existing Flight Recording` would be OK.